plaidml/edsl/tests/CPPLINT.cfg

```diff
@@ -1 +1 @@
-filter=-whitespace/line_length
+filter=-whitespace/line_length, -readability/braces
```

Comment:
Because we are using a macro `PLAIDML_EDSL_LOOP`, cpplint complains about the `;` after the loop body, but it is needed, since it is the end of a lambda function.

Reply:
Makes sense; the current approach seems good to me then.

Reply:
What is the purpose of the macro? In general I'd like to avoid macros unless absolutely necessary.

Reply:
The macro is used to 
1. for users to easier to write complex nested loops, and 
2. force users to pass a lambda function with `capture-default` as `[=]`

I think these are not absolutely necessary. I will change it to the form you mentioned below,
```
Tensor O = loop(5, A, [&](Tensor iv) {
  return iv + 1;
};)
```